packages/amplify-category-api/src/provider-utils/awscloudformation/service-walkthroughs/apigw-walkthrough.ts

```diff
@@ -830,12 +664,7 @@ export const openConsole = async (context: $TSContext) => {
     let selectedApi = restApis[0];
 
     if (restApis.length > 1) {
-      ({ selectedApi } = await inquirer.prompt({
-        type: 'list',
-        name: 'selectedApi',
-        choices: restApis,
-        message: 'Please select the API',
-      }));
+      selectedApi = await prompter.pick<'one', string>('Select the API', restApis);
     }
```

Comment:
nit: prompter.pick will automatically skip the prompt and return the single item if there is only one option